designcompose/src/main/java/com/android/designcompose/squoosh/SquooshAnimate.kt

```diff
@@ -462,6 +473,10 @@ private fun needsStyleTween(a: ViewStyle, b: ViewStyle): Boolean {
     return false
 }
 
+private fun needsOpacityTween(a: ViewStyle, b: ViewStyle): Boolean {
+    return a.node_style.opacity != b.node_style.opacity
```

Comment:
super nit: should we compare opacity.getElse(1f)?

Reply:
Good idea, that would avoid animations between opacity 1 and opacity Optional.NULL

Reply:
> super nit: should we compare opacity.getElse(1f)?

+1, because `opacity.getOrElse { 1F }` is used for when `needsOpacityTween==true`.

Reply:
done